getting-started/what-you-need.md

```diff
@@ -1,102 +1,116 @@
 
+## Adding the Client Assets
 
-## Client Assets
+For the Telerik UI for Blazor components to look and behave as expected in the browser, add the [Telerik stylesheet](#telerik-stylesheet) and the [JS Interop file](#telerik-js-interop-file) to your main index file. Depending on the Blazor hosting model and framework version, your project's main index file will differ:
 
-To have the Telerik Blazor components look and behave as expected in the browser, you need some assets. 
-@[template](/_contentTemplates/common/js-interop-file.md#app-paths)
+   * For Client-Side Blazor apps, use the `wwwroot/index.html` index file.
+   * For Server-Side Blazor apps, use one of the following index files:
+      * `~/Pages/_Host.cshtml` for .NET 3.x and .NET 5
+      * `~/Pages/_Layout.cshtml` for .NET 6
 
+### Telerik Stylesheet
 
-#### The Telerik stylesheet - For more information, refer to the [Themes]({%slug general-information/themes%}) article. Here is a short example:
+The stylesheet allows you to use one of the built-in [Themes]({%slug general-information/themes%}), for example, the Default theme:
 
 @[template](/_contentTemplates/common/js-interop-file.md#theme-static-asset-snippet)
 
 
-#### The Telerik JS Interop file - Provides features that cannot be implemented with native Blazor.
+### Telerik JS Interop File
+
+The JS Interop file provides features that cannot be implemented with native Blazor.
 
 @[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
 
 
-### Static Assets
+### Using Static Assets
 
-You can add the Telerik JS Interop file as a [static asset](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to reference the asset as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
+You can add the [Telerik JS Interop file](#telerik-js-interop-file) as a [static asset](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to reference the asset as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
 
-To enable the use of static assets in your project, add the following line to the startup file of your **Server** project:
+To enable the use of static assets in your project, add the `app.UseStaticFiles();` line to the startup file of your **Server** project:
```

Comment:
This line is present by default in the default Visual Studio project template. The developer just has to make sure it's still there.